packages/tasks/src/model-libraries.ts

```diff
@@ -214,6 +214,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			term: { path: "gliner_config.json" },
 		},
 	},
+	"glyph-byt5": {
+		prettyLabel: "Glyph-ByT5",
+		repoName: "Glyph-ByT5",
+		repoUrl: "https://github.com/AIGText/Glyph-ByT5",
+		filter: false,
+		countDownloads: {
+			term: { path: "checkpoints/byt5_model.pt" },
```

Comment:
Overall LGTM. Just note that we strongly suggest using conventions that will make it easier to scale up. For example, if you change the checkpoint name, this query won't work. To handle this, we usually use a file such as `config.json` or another query file that will always be there for all repos under a library. Would you consider doing something like this for the glyph repos?

Reply:
Thanks for the advice! We would consider such a format in the future, but at the moment we would prefer keeping the current format.

Reply:
SG!